server/routes/api.js

```diff
@@ -9601,4 +9601,322 @@ router.get('/getClinicEmployees/:id', function (req, res) {
 
 /* End Registered Clinics */
 
+// start email drafts
+
+router.get("/getEmailDrafts/:id", function (req, res, next) {
+  connection.getConnection(function (err, conn) {
+    if (err) {
+      logger.log("error", err.sql + ". " + err.sqlMessage);
+      res.json(err);
+    }
+    var id = req.params.id;
+    conn.query(
+      "select * from email_drafts where superadmin = ?",
+      [id],
+      function (err, rows) {
+        conn.release();
+        if (!err) {
+          res.json(rows);
+        } else {
+          res.json(err);
+          logger.log("error", err.sql + ". " + err.sqlMessage);
+        }
+      }
+    );
+  });
+});
+
+router.post("/createEmailDraft", function (req, res, next) {
+  connection.getConnection(function (err, conn) {
+    if (err) {
+      logger.log("error", err.sql + ". " + err.sqlMessage);
+      res.json(err);
+    }
+
+    response = null;
+    logger.log("req.body: ", req.body)
+    var data = {
+      birthdayFrom: req.body.birthdayFrom,
```

Comment:
We have here an identical object with the same property.
Why do we additional mapped property?
For example, if we need to add a new property in this way we need to change it in three places (front, back, and database).
Why we didn't create SQL query and send a directly req.body object?
**insert email_drafts set ? where id = '" + req.body.id + "'", req,body**
This also applies to other methods